xlators/cluster/dht/src/dht-rebalance.c

```diff
@@ -3532,7 +3532,7 @@ gf_defrag_process_dir(xlator_t *this, gf_defrag_info_t *defrag, loc_t *loc,
     return ret;
 }
 
-int
+static int
```

Comment:
nit: considering this is a static function, called from 2 places, are lines 3553-3558 needed?